### PR TITLE
driver: fix overwriting error in NodePublishVolume

### PR DESCRIPTION
**Pull request by ymmt2005**
https://topolvm.slack.com/archives/CT0AS6UQ1/p1620226390020100

### CODE REVIEW

File: driver/node.go [hunk 1]
```diff
@@ -154,7 +154,7 @@ func (s *nodeService) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 			// guarantee that NodePublishVolume will be called again, so if
 			// anything fails after the volume is created we need to attempt to
 			// clean up the LVM so we don't leak storage space.
-			if _, err = s.lvService.RemoveLV(ctx, &proto.RemoveLVRequest{Name: volumeID, DeviceClass: topolvm.DefaultDeviceClassName}); err != nil {
+			if _, err := s.lvService.RemoveLV(ctx, &proto.RemoveLVRequest{Name: volumeID, DeviceClass: topolvm.DefaultDeviceClassName}); err != nil {
```
Comment (ymmt2005) [inline driver/node.go:157]: This.

Comment (ymmt2005) [inline driver/node.go:157]: When this goes successful, the line 161 would return `nil, nil` and cause the error.

**Review by toshipp — APPROVED** [review]
LGTM

**Review by llamerada-jp — APPROVED** [review]
LGTM

**Pull request merged**